qualle/interface/cli.py

```diff
@@ -179,13 +180,25 @@ def cli_entrypoint():
     )
     train_parser.set_defaults(func=handle_train)
     train_parser.add_argument(
-        "train_data_path",
-        type=str,
-        help="Path to train data. "
+        "predict_train_data_path",
+        type=Path,
+        help="Path to train data for the Recall Predictor. "
         "Accepted inputs are either a tsv file or "
         "a folder in Annif format.",
```

Comment:
Maybe you already answered that for stwfsapy, just wondering again: Do people know what THE Annif format ist? Otherwise, can we include a link here?

Reply:
The tsv and Annif format are explained in the README file. Perhaps we should reference that?

"For more information about the input format, see the README file at https://github.com/zbw/qualle."

Reply:
Yes, that would do.